src/Components/FirmwareSelector/__tests__/index.test.jsx

```diff
@@ -235,7 +235,9 @@ describe('FirmwareSelector', () => {
     expect(screen.getByText('selectTarget (Display Name)')).toBeInTheDocument();
   });
 
-  it('should allow changing firmware options for AM32', async() => {
+
+  /** no we dont want to change firmware anymore */
```

Comment:
Then we should make sure that it's not possible to change it anymore ;-)

Reply:
i am wrong here, we need to select firmware from older versions than 1.93